src/stylesheets/test/tests.scss

```diff
@@ -1,4 +1,10 @@
-$_error-output-override: true;
+@use "../core/functions/general/error.scss" with (
+  $_error-output-override: true
+);
+
+@use "../core/settings/settings-general" with (
+  $theme-show-compile-warnings: true
+);
```

Comment:
This is a good workaround. We explore how to make this work the classic way to avoid conflicts when updating.

```scss

$_error-output-override: true;
$theme-show-compile-warnings: true;

$theme-show-notifications: false;
$theme-color-primary-lightest: "gray-warm-3";
$theme-color-warning: "orange-50v";
$theme-color-secondary: $custom-hex-grade-65;
$theme-color-secondary-vivid: $custom-hex-grade-55;
$theme-color-secondary-dark: $custom-hex-grade-70;
$theme-color-accent-warm-lightest: $custom-hex-grade-25;
$theme-color-accent-cool-lightest: false;

$theme-link-color: "primary";
$theme-link-visited-color: "violet-70v";
$theme-link-hover-color: "primary-dark";
$theme-link-active-color: "primary-darker";
$theme-link-reverse-color: "white";
$theme-link-reverse-hover-color: "gold-20v";
$theme-link-reverse-active-color: "white";

@import "../uswds";
```